examples/QIR/Development/README.md

```diff
@@ -8,3 +8,31 @@ Note: If you are using the Development project on Linux, you may run into the fo
 ```
 export LD_LIBRARY_PATH=/usr/lib/llvm-11/lib
 ```
+
+# Development of the QIR Debug Info Generation
+NOTE: This debug feature is currently incompatible with Windows due to some LLVM tool limitations. If you request debug symbols within Development.csproj while using the Windows platform, you will run into errors when you build the Development project. These errors come from compiling the emitted QIR (with debug information) into an executable, meaning that the QIR emission itself will still work on Windows, but you won't be able to actually test the Q# debugging experience.
```

Comment:
Is this due to the problem with the .qs extension on Windows? If so, can you please expand this note adding a brief couple of sentences explaining what problem would someone run into when trying to test the new Q# debugging experience?

Reply:
Yes, that's what it's related to. I wrote a bit more here.